### PR TITLE
Ticket 5

### PR DESCRIPTION
**Pull request by Cartesianist**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -52,22 +45,45 @@ describe('/api/articles/:article_id', () => {
...
-        })
-    })
-})
\ No newline at end of file
+                });
+        });
+    });
+});
+
+describe("/api/Users", () => {
```
Comment (HMelesi) [inline __tests__/app.test.js]: I'd remove the capital U here!

File: __tests__/app.test.js [hunk 2]
```diff
@@ -52,22 +45,45 @@ describe('/api/articles/:article_id', () => {
...
+    describe('GET, 200: returns all Users', () => {
+        test('returns an array of 4 objects with the expected keys', () => {
+            return request(app)
+                .get('/api/Users')
```
Comment (HMelesi) [inline __tests__/app.test.js]: And here... It should work anyway but just for consistency

**Review by HMelesi — APPROVED** [review]
Great work Ibraheem! Just correct that uppercase U to be a lowercase u and it's all approved, no need to resubmit the PR.

File: __tests__/app.test.js (hunk 1 above)
Comment (Cartesianist) [reply __tests__/app.test.js]: Thanks!

**Pull request merged**